.travis.yml

```diff
@@ -2,3 +2,5 @@ osx_image: xcode10.2
 language: objective-c
 script:
 - "./ios/travis-ci/build-ipa.sh"
+after_script:
+  - sleep 10
```

Comment:
How did you come up with 10?

Reply:
it was the value suggested by support, I didnt think about it much